### PR TITLE
jq

### PR DESCRIPTION
**Pull request by clay584**
## Status
Ready

## Related Content Pull Request
Related PR: https://github.com/demisto/content/pull/14454

## Related Issues
Related:

## Description
This is a container that has jq compiled, installed, and the pyjq python package installed for use with a transformer script for XSOAR.

### CODE REVIEW

File: docker/jq/verify.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    id="2013-12-27_00-09-37",
+    changeSet=dict(items=[], kind="git"),
+)
+
+d = pyjq.first('.parameters[] | {"param_name": .name, "param_type":.type}', data)
+
+try:
+    assert d == golden_data
+except AssertionError:
+    sys.exit(1)
```
Comment (DeanArbel) [inline docker/jq/verify.py:23]: Why do you `except` the assert?

**Review by DeanArbel — APPROVED** [review]
Looks good

**Comment by clay584** [comment]
I wasn’t sure if your CI would bomb out if there was an actual exception.

**Comment by DeanArbel** [comment]
> I wasn’t sure if your CI would bomb out if there was an actual exception.
That's ok - if it doesn't work as intended it's ok for it to "bomb out".

**Comment by tallieber** [comment]
So @clay584 can you remove the try-except clauses?

**Comment by clay584** [comment]
Thank you for your patience and working with me. This is one of the first times I have contributed to XSOAR outside of just submitting something in the UI.

**Pull request merged**